vendor/bat-native-ads/src/bat/ads/internal/base/strings/string_html_parse_util.cc

```diff
@@ -0,0 +1,40 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <regex>
+
+#include "bat/ads/internal/base/strings/string_html_parse_util.h"
```

Comment:
the linter might complain that this is included below the `<regex>` include

Reply:
Resolved